### PR TITLE
Update appending-nodes-to-the-document.md

### PR DESCRIPTION
**Pull request by nem035**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/javascript/browser-apis/dom-manipulation/appending-nodes-to-the-document.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/browser-apis/dom-manipulation/appending-nodes-to-the-document.md?ref=33e5073c08d09b644c0e0b4491b35d9bf7316582)

File: javascript/browser-apis/dom-manipulation/appending-nodes-to-the-document.md [hunk 1]
```diff
@@ -37,25 +23,26 @@ Once you have created an element you need to add it to the HTML document, or it
...
+);
 
 function initializeElement(element, text) {
-  var newElement = document
-    .createElement(element)
-  newElement
-   .appendChild(document.
-      createTextNode(text))
-  return newElement
+  var newElement = document.createElement(
```
Comment (kapnobatai137) [inline javascript/browser-apis/dom-manipulation/appending-nodes-to-the-document.md]:
```suggestion
  let newElement = document.createElement(
```
shoud we update this `var` to `let`?

File: javascript/browser-apis/dom-manipulation/appending-nodes-to-the-document.md [hunk 2]
```diff
@@ -77,12 +64,12 @@ After creating an element, which method is used to add the element to the HTML d
...
 
 ```javascript
-var container = document
-  .getElementById('container');
-var paragraph = document
-  .createElement('p');
+var container = document.getElementById(
+  "container"
+);
+var paragraph = document.createElement("p");
```
Comment (kapnobatai137) [inline javascript/browser-apis/dom-manipulation/appending-nodes-to-the-document.md]:
```suggestion
let container = document.getElementById(
  "container"
);
let paragraph = document.createElement("p");
```

same here, should we update to `let`?

File: javascript/browser-apis/dom-manipulation/appending-nodes-to-the-document.md (hunk 1 above)
Comment (nem035) [reply javascript/browser-apis/dom-manipulation/appending-nodes-to-the-document.md]: Yeah, why not.

The rule can be, always use `let`, unless to demonstrate the nuances of `var` or `const`

**Pull request merged**